modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -46,8 +46,8 @@ class NDB_Form_statistics extends NDB_Form
             $subcat  = $row['Subcat'];
             $center  = $row['CenterID'];
 
-            if(array_search($vl, $visits) !== false
-                && array_search($subcat, $subcats) !== false
+            if(in_array($vl, $visits) !== false
```

Comment:
The `!== false` look like it means `true` which is exactly what is return by `true === true`.
Can we remove the `!== false`? in_array and _inCenter both returns a boolean.